src/main/java/data/JsonConverter.java

```diff
@@ -24,36 +24,43 @@
 /** A class that creates a config file from a Vendor object. */
 public class JsonConverter {
   private static final String FILE_PATH_BASE = "data/";
-  
-  public File createFile(String vendorID) {
-    String fileName = FILE_PATH_BASE + vendorID;
 
-    // Create a JSON object to write to the file.
-    JSONObject parentObject = new JSONObject();
-    JSONObject JSONCar = new JSONObject();
-    JSONCar.put("Car", "Blue Tacoma");
-    parentObject.put("Vehicle", JSONCar);
+  public boolean updateFile(Vendor vendor) {
+    String jsonConfig = vendor.createConfig();
+
+    // Create and write the contents to a File.
+    File billingFile = buildFile(vendor.getVendorID(), jsonConfig);
+    
+    if(billingFile == null) {
+      return false;
+    }
+    return true;
+  }
 
-    // Write the JSON object to the file.
-    File file = null;
+  /**
+   * Create a new billig config file.
+   * @param vendorID a String representing a Vendor's ID.
+   * @param jsonConfig a String filled with billing config content.
+   * @return File representing a newly built config file.
+   */
+  public File buildFile(String vendorID, String jsonConfig) {
+    File billingFile = null;
     try {
-      file = new File(fileName);
-      BufferedWriter out = new BufferedWriter(new FileWriter(file));
-      out.write(parentObject.toString());
+      billingFile = new File(FILE_PATH_BASE + vendorID);
+      BufferedWriter out = new BufferedWriter(new FileWriter(billingFile));
+      out.write(jsonConfig);
       out.close();
     } catch(IOException e) {
       return null;
```

Comment:
I don't like the idea of returning null here. Instead, why not throw an exception and you can catch it elsewhere instead of doing a null check

Reply:
+1, talk to Cade about what he would prefer from this API (since he is the customer).
Okay to address in follow-up

Reply:
Sounds good, will do.